d3/d3.d.ts

```diff
@@ -119,7 +119,7 @@ declare module D3 {
         * @param arr Array to search
         * @param map Accsessor function
         */
-        extent<T, U>(arr: T[], map: (v: T) => U): U[];
+        extent<T, U>(arr: T[], map: (v: T, i?: number) => U): U[];
```

Comment:
According to the source code below, I think the map function has only two parameter.
https://github.com/mbostock/d3/blob/master/src/arrays/extent.js#l14#l15
I noted that all the parameters of map function of `max` and `min` are optional also. But I can't understand, what's the reality meaning of it? If the first parameter is missed, these three method do not work, aren't they?